elastic4s-tests/src/test/scala/com/sksamuel/elastic4s/requests/mappings/MappingHttpTest.scala

```diff
@@ -74,5 +84,55 @@ class MappingHttpTest extends WordSpec with DockerTests with Matchers with Befor
 
       properties shouldBe Map.empty
     }
+
+    "handle properly completely empty mapping" in {
+
+      val mappings = client.execute {
+        getMapping("indexnopropsempty")
+      }.await.result
+
+      val properties = mappings.find(_.index == "indexnopropsempty").get.mappings
+
+      properties shouldBe Map.empty
+    }
+  }
+
+  "field mapping get" should {
```

Comment:
Test cases for when we ask for a specific field in a mapping get request. These are a mirror of the original two tests in addition to the third one I added for when we get all mappings.